libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -696,39 +697,50 @@ open class LoginActivity : AppCompatActivity(), OAuthWebviewHelperEvents {
     // region QR Code Login Via UI Bridge API Public Implementation
 
     /**
-     * Automatically log in with a UI Bridge API front door bridge URL and PKCE code verifier.
+     * Automatically log in with a UI Bridge API front door bridge URL and PKCE
+     * code verifier.
      *
-     * This method is the intended entry point to Salesforce Mobile SDK when using the Salesforce
-     * Identity API UI Bridge front door URL.  Usable, default implementations of methods are
-     * provided for parsing the UI Bridge parameters from the reference JSON and log in URLs used
-     * by the reference QR Code Log In implementation.  However, the URL and JSON structure in the
-     * reference implementation is not required.  An app may use a custom structure so long as this
-     * entry point is used to log in with the front door URL and optional PKCE code verifier.
+     * This method is the intended entry point to Salesforce Mobile SDK when
+     * using the Salesforce Identity API UI Bridge front door URL.  Usable,
+     * default implementations of methods are provided for parsing the UI Bridge
+     * parameters from the reference JSON and log in URLs used by the reference
+     * QR Code Log In implementation.  However, the URL and JSON structure in
+     * the reference implementation is not required.  An app may use a custom
+     * structure so long as this entry point is used to log in with the front
+     * door URL and optional PKCE code verifier.
      *
      * @param frontdoorBridgeUrl The UI Bridge API front door bridge URL
-     * @param pkceCodeVerifier The optional PKCE code verifier, which is not required for User Agent
+     * @param pkceCodeVerifier The optional PKCE code verifier, which is not
+     * required for User Agent
      * Authorization Flow but is required for Web Server Authorization Flow
      */
     @Suppress("MemberVisibilityCanBePrivate")
     fun loginWithFrontdoorBridgeUrl(
         frontdoorBridgeUrl: String,
         pkceCodeVerifier: String?
-    ) = webviewHelper?.loginWithFrontdoorBridgeUrl(frontdoorBridgeUrl, pkceCodeVerifier)
+    ) {
+        SalesforceSDKManager.getInstance().registerUsedAppFeature(FEATURE_QR_CODE_LOGIN)
```

Comment:
I think we should move this up one call to `loginWithFrontdoorBridgeUrlFromQrCode` so we don't miss-report QR usage if  `loginWithFrontdoorBridgeUrl` is used for another flow.